Misc/NEWS.d/next/Library/2023-08-29-22-53-48.gh-issue-108590.6k0pOl.rst

```diff
@@ -0,0 +1 @@
+Fixed an issue causing sqlite3.iterdump to crash when encountering invalid unicode in a TEXT column.
```

Comment:
"Crash" is normally used for coredumps only. Python exceptions are "failures" or "exceptions raised", or similar. Also, it helps if the NEWS entry explains the user visible effect of the failure (in this case: an incomplete or faulty SQL dump is produced). How about this?

```suggestion
Fixed an issue where :meth:`sqlite3.Connection.iterdump` would fail and leave an incomplete SQL dump if a table includes invalid Unicode sequences.
```

Reply:
Also, you may append "Patch by \<your name\>" at the end of the NEWS entry.